lookout/core/lib.py

```diff
@@ -82,11 +82,14 @@ def files_by_language(files: Iterable[File]) -> Dict[str, Dict[str, File]]:
     :param files: Iterable of `File`-s.
     :return: Dictionary with languages as keys and files mapped to paths as values.
     """
-    result = defaultdict(dict)
-    for file in files:
+    result = OrderedDict()
+    for file in sorted(files, key=lambda x: x.path.lower()):
```

Comment:
the `.lower()` here might in some rare cases create non-determinism (`/home/mog/A` and `/home/mog/a` could be returned in different orders). It's probably safer to remove it.

Reply:
ok thanks !